app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/recent/RecentRecyclerAdapter.java

```diff
@@ -2,36 +2,29 @@
 
 import android.content.Context;
 import android.support.v7.widget.RecyclerView;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports